### PR TITLE
[SQL] `az sql midb ledger-digest-uploads`: Support SQL Ledger

### PR DESCRIPTION
**Pull request by paramsiva1**
**Related command**

`az sql midb create`
`az sql midb ledger-digest-uploads`

**Description**

Adding Azure CLI commands for Ledger in MI (az sql midb ledger-digest-uploads).
Adding parameter to az sql midb create for ledger database

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [x] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
MI

File: src/azure-cli/azure/cli/command_modules/sql/_help.py [hunk 1]
```diff
@@ -1282,6 +1282,35 @@
     text: az sql midb log-replay wait --deleted --resource-group mygroup --managed-instance myinstance --name mymanageddb
 """
 
+helps['sql midb ledger-digest-uploads'] = """
```
Comment (evelyn-ys) [inline src/azure-cli/azure/cli/command_modules/sql/_help.py:1285]:
```suggestion
helps['sql midb ledger-digest-upload'] = """
```

**Review by evelyn-ys — APPROVED** [review]
Please add tests for new commands

File: src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py [hunk 2]
```diff
@@ -5746,6 +5746,49 @@ def test_sql_managed_db_mgmt(self, mi, rg):
                  .format(resource_group_1, managed_instance_name_1, database_name),
                  expect_failure=True)
 
+    @ManagedInstancePreparer()
+    def test_sql_midb_ledger(self, mi, rg):
```
Comment (evelyn-ys) [inline src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py:5750]: After adding test, pls commit the test recording file too.

Comment (paramsiva1) [reply src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py:5750]: Added test recordings.

File: src/azure-cli/azure/cli/command_modules/sql/_help.py (hunk 1 above)
Comment (paramsiva1) [reply src/azure-cli/azure/cli/command_modules/sql/_help.py:1285]: we use same cmd in azure sql also. So, using the same to maintain the consistency between azure sql and MI.
https://learn.microsoft.com/en-us/cli/azure/sql/db/ledger-digest-uploads?view=azure-cli-latest#az-sql-db-ledger-digest-uploads-enable

**Pull request merged**